staticaddr/address/log.go

```diff
@@ -0,0 +1,29 @@
+package address
```

Comment:
Another approach to subpackage logging could be to just have one logger in the `staticaddr` package and then each subpackage just defines a global variable (`log`) which we assign with a modified `UseLogger` function.

Reply:
That reminds me that I didn't want to clutter the logs too much with subsystem logs. I think we can just summarize all deposit/withdraw/loop-in operations inside the `SADDR` logger. I think the log becomes easier to read then.

Reply:
I placed a single logger into the `staticaddr` package. For the loop-in I might add a separate logger, but creation/deposits/withdrawals I think can be under the same sublogger.